src/main/resources/application.yaml

```diff
@@ -115,6 +116,8 @@ quarkus:
         level: ERROR
       'com.datastax.oss.driver':
         level: WARN
+      'io.quarkus.http.access-log':
```

Comment:
This is important: with default level of WARN request/access log would not be included without.